api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -775,6 +775,19 @@ message AccessGraphAWSScanEvent {
   uint64 total_accounts = 11;
 }
 
+// AccessGraphAzureScanEvent is emitted when the Access Graph
+// Azure scan is enabled.
+message AccessGraphAzureScanEvent {
+  // total_vms is the total amount of virtual machines found in the Azure scan.
```

Comment:
lets exclude the usage events for now. It requires PRs to cloud repo before.

Everything else looks fine